org.kde.artikulate.json

```
@@ -0,0 +1,32 @@
+{
+  "id": "org.kde.artikulate",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15",
+  "sdk": "org.kde.Sdk",
+  "command": "artikulate",
+  "rename-icon": "artikulate",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
```

```suggestion
    "--socket=wayland",
    "--socket=pulseaudio",
```

Otherwise playing sounds fails:

```
Warning: "Failed to connect: Connection refused"
log_artikulate: Player(0x558690f9e470) Playback sound in file  "/var/home/bpiotrowski/.var/app/org.kde.artikulate/data/artikulate/courses/basic/fr/{d1e2b44d-2356-405a-b1dc-cebfab666a16}.ogg"
Warning: "Failed to connect: Connection refused"
```